### PR TITLE
Add search bar

### PR DESCRIPTION
**Pull request by martinhaiss**
This resolves [#9](https://github.com/martinhaiss/capstone-project/projects/1#card-71894983)

I implemented a search bar into the header, so users can search for matching activities by postal code or city.

### CODE REVIEW

File: src/components/SearchBar.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <label>
+        <ScreenReaderInfo>Search bar</ScreenReaderInfo>
+        <Input
+          type="text"
+          name="input"
+          placeholder="Suche nach PLZ oder Stadt"
+          isToggled={isToggled}
```
Comment (macfrei) [inline src/components/SearchBar.js:23]: What does `isToggled` do?

**Review by macfrei — COMMENTED** [review]
Nice job, just added a few comments :)

File: src/components/Card.js [hunk 2]
```diff
@@ -41,7 +41,10 @@ function ActivityCard({
             onDeleteButtonClick(id);
           }}
         >
-          <DeleteIcon style={{ width: "20", height: "20" }} aria-label="home" />
+          <DeleteIcon
+            style={{ width: "20", height: "20" }}
+            aria-label="delete"
```
[the remark this answers is not in the document]
Comment (martinhaiss) [reply src/components/Card.js:46]: Fixed! 🙂

File: src/components/SearchBar.js (hunk 1 above)
Comment (martinhaiss) [reply src/components/SearchBar.js:23]: Nothing 🙈 It's a holdover from my first version, when I was working with a button + icon. I forgot to delete the code after deciding to go just with an input field.

File: src/App.js [hunk 3]
```diff
@@ -13,12 +13,17 @@ import CreateActivity from "./components/CreateActivity";
...
+import searchActivities from "./lib/searchActivities";
 
 function App({ data }) {
   const [activities, setActivities] = useState(
     loadFromLocal("localActivities") ?? data
   );
 
+  const [searchTerm, setSearchTerm] = useState("");
+  const searchedActivities =
+    searchActivities(activities, searchTerm) || activities;
```
[the remark this answers is not in the document]
Comment (martinhaiss) [reply src/App.js:25]: Thx for the tip!

**Pull request merged**